apps/proposer/src/services/proposer.mjs

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable import/no-unresolved */
+import submitTransaction from '../utils/submitTransaction.mjs';
+import { BadRequestError } from '../../common-files/utils/errors.mjs';
+import { address } from '../classes/keys.mjs';
+import { optimist } from '../classes/http.mjs';
+import logger from '../../common-files/utils/logger.mjs';
+
+export async function getCurrentProposer() {
+  const { currentProposer } = await optimist.get(`/proposer/current-proposer`);
+  return currentProposer;
+}
+
+export function getProposers() {
+  const res = optimist.get(`/proposer/proposers`);
+  return res;
+}
+
+export async function registerProposer(bond) {
```

Comment:
We could add validation to the parameters using the library being used in `sdk`, I think the name is `joi`, raising BadRequestError when a validation fails.